continuous_integration/test_script.sh

```diff
@@ -29,8 +31,8 @@ else
 fi
 
 # Is directory still empty ?
-ls
+ls -ltra
```

Comment:
@arthurmensch not sure what is the point of this, would you mind elaborating on this, i.e. why do we want the dir to be empty?